packages/webr/src/mount.c

```diff
@@ -0,0 +1,121 @@
+#define R_NO_REMAP
+
+#include <R.h>
+#include <Rinternals.h>
+
+#ifdef __EMSCRIPTEN__
+#include <emscripten.h>
+#endif
+
+SEXP ffi_mount_workerfs(SEXP base_url, SEXP mountpoint) {
+#ifdef __EMSCRIPTEN__
+  if (!Rf_isString(base_url) || LENGTH(base_url) != 1) {
+    Rf_error("URL must be a character string.");
+  }
+  
+  if (STRING_ELT(base_url, 0) == NA_STRING){
+    Rf_error("URL must not be `NA`.");
+  }
+
+  if (!Rf_isString(mountpoint) || LENGTH(mountpoint) != 1) {
+    Rf_error("`mountpoint` must be a character string.");
+  }
+  
+  if (STRING_ELT(mountpoint, 0) == NA_STRING){
+    Rf_error("`mountpoint` must not be `NA`.");
+  }
+
+  EM_ASM({
+    const baseUrl = UTF8ToString($0);
+    const mountpoint = UTF8ToString($1);
+    const dataResp = Module.downloadFileContent(`${baseUrl}.data`);
+    const metaResp = Module.downloadFileContent(`${baseUrl}.js.metadata`);
+    if (dataResp.status < 200 || dataResp.status >= 300 ||
```

Comment:
Create paragraphs to give some breathing room?

```suggestion

    if (dataResp.status < 200 || dataResp.status >= 300 ||
```